src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * 
+ * A subclass of the Task Object. 
+ * Includes a deadline date to complete task by.
+ * 
+ */
+public class Deadline extends Task {
+
+    protected LocalDate date;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.date = LocalDate.parse(by);
+    }
+
+    public Deadline(String description, boolean isDone, String by) {
+        super(description, isDone);
+        this.date = LocalDate.parse(by);
+    }
```

Comment:
LocalDate.parse can throw DateTimeException. Probably should handle it here or put it in constructor definition.
```suggestion
    public Deadline(String description, String by) throws DateTimeException {
        super(description);
        this.date = LocalDate.parse(by);
    }

    public Deadline(String description, boolean isDone, String by) throws DateTimeException {
        super(description, isDone);
        this.date = LocalDate.parse(by);
    }
```